.claude/skills/radio-link-testing/scripts/tier1-patch.sh

```
@@ -0,0 +1,94 @@
+echo "[tier1] parts verified; $remaining still bad after $((MAXR+1)) rounds; deliver-time=$(( $(now)-GRAND0 ))s"
+[ "$remaining" -ne 0 ] && { echo "[tier1] ABORT: parts still bad"; exit 1; }
+
+# --- reassemble in order: clear target, append each part ---
+ASM=//tp_asm.bin
+echo "[tier1] === reassembling -> $ASM ==="
+RM_FILE="$ASM" SPRAY_N=8 "${PYTEST[@]}" "$T::test_remove_file" >/dev/null 2>&1; sleep 1
+tasm=$(now)
+for i in "${!parts[@]}"; do
+  mark=$(wc -l <"$LOG")
+  # AppendFile is NOT idempotent -> send ONCE, confirm AppendFileSucceeded (retry only if not seen)
+  for tries in 1 2 3; do
```

Shellcheck (0.11.0)

[warning] tries appears unused. Verify use (or export if used externally).

(SC2034)